.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - nightly
+  #- nightly
```

Comment:
Might be worth using `if [ $TRAVIS_RUST_VERSION == "stable"]` on rustfmt rather than just disabling nightly?

Reply:
rustc warns about things in the generated jsapi bindings. When these don't have rustfmt applied to them, the entire contents of the file appears on one line which exceeds the travisci log limit and the build fails. We can't build without rustfmt, sadly.

Reply:
Oh that's annoying!